app/src/main/java/com/github/libretube/ui/sheets/PlaylistOptionsBottomSheet.kt

```diff
@@ -105,8 +114,21 @@ class PlaylistOptionsBottomSheet(
                 }
 
                 getString(R.string.change_playlist_description) -> {
-                    PlaylistDescriptionDialog(playlistId, "", onChangeDescription)
-                        .show(parentFragmentManager, null)
+                    val bundle = bundleOf(
+                        IntentData.playlistId to playlistId,
+                        IntentData.playlistDescription to ""
+                    )
+                    val newShareDialog = PlaylistDescriptionDialog()
+                    newShareDialog.arguments = bundle
+                    newShareDialog.show(parentFragmentManager, null)
+                    parentFragmentManager.setFragmentResultListener(
```

Comment:
I think we should create the listener before we show the dialog?

Reply:
It partially works. new description gets displayed only when the change description dialog is launched again by user.

Reply:
I got it working. I attached the fragment listener in the adapter, in this case the `PlaylistAdapter`.
 
`holder.binding.apply { `
` //some other code`
`val fragmentManager = (root.context as BaseActivity).supportFragmentManager`
` fragmentManager.setFragmentResultListener(`
`                IntentData.requestKey,`
`               (root.context as BaseActivity) `-> is this correct LifecycleOwner for it according to you?
`           ) { _, resultBundle ->`

Reply:
Seems fine if it works :)

Reply:
Yes it works.